docs/website/data/integrations.yaml

```diff
@@ -1356,6 +1356,30 @@ integrations:
     videos:
       - https://www.youtube.com/watch?v=RJkgmdjJn_w
 
+  topaz:
+    title: Topaz
+    description:  |
+      Topaz is an open source authorization service providing fine grained, real-time, policy based access control for applications and APIs. 
+      Topaz combines the best of OPA and Zanzibar in a single container that you can host as a sidecar or microservice.
+    labels:
+      category: authorization
+      layer: application
+      type: poweredbyopa
+    inventors:
+      - aserto
+    software:
+      - topaz
+    code:
+      - https://github.com/aserto-dev/topaz
+    tutorials:
+      - https://www.topaz.sh
+      - https://github.com/aserto-dev/topaz#quickstart
+      - https://www.topaz.sh/docs/intro
```

Comment:
Let's not repeat it -- Let's add `https://www.topaz.sh` or `https://www.topaz.sh/docs/intro`, not both.

Also the quickstart section on github is identical to https://www.topaz.sh/docs/getting-started, so I don't think the repo needs to be linked again .here.

Reply:
removed the intro docs, left the microsite and the quickstart (which are different)